acceptance/tests/facts/windows.rb

```diff
@@ -22,7 +22,10 @@
     os_version  = '2012 R2'
     kernel_version = '6.3'
   elsif agent['platform'] =~ /-10/
-    os_version  = /^10\./
+    os_version  = '10'
```

Comment:
What do older versions of Windows look like under Facter 2? Could the above output be an artifact of not updating special logic for new versions of windows, same as it was in Facter 3?